suite-native/app/src/hooks/useCoinEnablingInitialCheck.tsx

```diff
@@ -42,6 +42,7 @@ export const useCoinEnablingInitialCheck = () => {
             let timeoutId: TimerId;
             //if btc only device, just run discovery for btc and do not show the UI
             if (hasBitcoinOnlyFirmware) {
+                // TODO: Does this line now make sense?
                 dispatch(setEnabledDiscoveryNetworkSymbols(['btc']));
```

Comment:
Imo yes. Otherwise if you have universal fw, skipping this would mean, that you wont see anything on dashboard and you just tried to fix that.

Reply:
As discussed in person, moved to `discoveryMiddleware`.